x-pack/plugins/ml/public/application/components/field_stats_flyout/field_stats_info_button.tsx

```diff
@@ -80,18 +80,28 @@ export const FieldStatsInfoButton = ({
           </EuiToolTip>
         ) : null}
       </EuiFlexItem>
-      <EuiFlexItem grow={false} css={{ paddingRight: themeVars.euiTheme.euiSizeXS }}>
+      <EuiFlexItem
+        grow={false}
+        css={{
+          paddingRight: themeVars.euiTheme.euiSizeXS,
+          paddingBottom: themeVars.euiTheme.euiSizeXS,
+        }}
+      >
         <FieldIcon
           color={isEmpty ? themeVars.euiTheme.euiColorDisabled : undefined}
           type={getKbnFieldIconType(field.type)}
           fill="none"
         />
       </EuiFlexItem>
-      <EuiFlexItem grow={true}>
-        <EuiText color={isEmpty ? 'subdued' : undefined} size="s">
-          {label}
-        </EuiText>
-      </EuiFlexItem>
+      <EuiText
+        color={isEmpty ? 'subdued' : undefined}
+        size="s"
+        aria-label={label}
+        className="euiComboBoxOption__content"
+        css={{ paddingBottom: themeVars.euiTheme.euiSizeXS }}
```

Comment:
I think it's worth also adding `title={label}` so we can show the full label on hover for long text which now gets truncated.